### ISSUE TITLE
Rebuild a tentacle's docs through the web UI

### ISSUE THREAD

**Issue by Gastove** [Gastove/doctopus#27]
You should be able to rebuild a tentacle's docs through the web ui

**Comment by jarofghosts** [comment]
on a similar note it would be good to have a way to hit an endpoint to do that similar to RTD

**Comment by Gastove** [comment]
Yeah, was thinking about that. Supporting webhooks isn't necessary per se -
but I think it'll be pretty easy once we have all the other bits.

**Comment by Gastove** [comment]
Now that #77 is in, I think we've got a way to do this. A `rebuild/tentacle-name` route can be exposed, either to the UI or to something like a web hook. We don't have a `rebuild-doc` function yet; the only thing we're lacking is a clean way to nuke all the old docs for a tentacle out of the DB.
